ConwaysGameOfLife.Tests/InputParserShould.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace ConwaysGameOfLife.Tests
+{
+    public class InputParserShould
+    {
+        [Theory]
+        [InlineData("0,0", 0, 0)]
+        [InlineData("-1,-1", -1, -1)]
+        [InlineData("100,8", 100, 8)]
+        public void WhenUserEntersStringOfOneAliveCellLocation_ReturnsListOfLocationContainingOnlyOneValue(string userInput, int x, int y) //TODO: shorten method name

```

Comment:
```suggestion
        public void ReturnListOfLocationsContainingOnlyOneValue_GivenAStringOfOneAliveCellLocation(string userInput, int x, int y)
```
The tests should make sense when read from the class name down e.g. `InputParserShould` `ReturnListOfLocationsContainingOnlyOneValue_Given...`